### ISSUE TITLE
ESM Resolution failing

### ISSUE THREAD

**Issue by dlarocque** [google/safevalues#385]
Error:
```
$ yarn build

> myapp@0.1.0 build
> react-scripts build

Creating an optimized production build...
Failed to compile.

Module not found: Error: Can't resolve './builders/html_builders' in '/private/tmp/myapp/node_modules/safevalues/dist/mjs'
Did you mean 'html_builders.js'?
BREAKING CHANGE: The request './builders/html_builders' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

Reproduction Steps:
1. `npx create-react-app repro-app --template typescript`
2. `cd repro-app`
3. `yarn add safevalues`
4. Modify `src/App.tsx` to be the following:
```ts
import React from 'react';
import logo from './logo.svg';
import './App.css';
import { sanitizeHtml } from 'safevalues';


function App() {
  const html = sanitizeHtml('<article>my post <script>alert(0)</script></article>');
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.tsx</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
    </div>
  );
}

export default App;
```
5. `yarn build`

**Comment by dlarocque** [comment]
It works when I modify the imports to have the .js file extensions.

**Comment by dlarocque** [comment]
Would it be possible to provide a bundled version of this library, so that there are no issues with module resolution in some environments?

**Comment by neuracr** [comment]
Thanks for reporting. I'll add the .js extensions to the path-based imports.

**Comment by neuracr** [comment]
I've released safevalues 0.6.0 on NPM. It includes https://github.com/google/safevalues/commit/fa5bbf0fda5605d26ba1daf5c6a7f809f817c0af which adds the `.js` extension to all the path-based imports.
